src_python/habitat_sim/utils/viz_utils.py

```diff
@@ -287,3 +287,27 @@ def semantic_to_rgb(semantic_image: np.ndarray) -> np.ndarray:
     semantic_image_rgb.putdata((semantic_image.flatten() % 40).astype(np.uint8))
     semantic_image_rgb = semantic_image_rgb.convert("RGBA")
     return semantic_image_rgb
+
+
+def get_island_colored_map(island_top_down_map_data: np.ndarray):
+    """
+    Get the topdown map for a scene with island colors.
+
+    :param island_top_down_map_data: The island index map data from Pathfinder.get_topdown_island_view()
+
+    :return: rgb Image of islands at the desired slice.
+    """
+
+    island_map = Image.new("RGB", island_top_down_map_data.shape)
+    pixels = island_map.load()
+    white = int("0xffffff", base=16)
+    for x in range(island_top_down_map_data.shape[0]):
```

Comment:
setting all the pixels to white is easily vectorizable and maybe faster to do before the loop?

Reply:
**Image.new** has a 3 argument overload where it takes (mode, size, color), where... 

> The colour is given as a single value for single-band images, and a tuple for multi-band images (with one value for each band). If the colour argument is omitted, the image is filled with black. If the colour is None, the image is not initialised.

That'd probably be better, no?

Reply:
Easier AND faster.